.github/workflows/test.yml

```diff
@@ -10,13 +15,56 @@ jobs:
   # test local kind start up
   test_localkind:
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        # I suppose we'd better update version pre kepler release.
+        # keep from current default version and latest version.
+        kubectl_version: [v1.25.4, v1.27.2]
+        kind_version: [0.17.0, 0.19.0]
+    steps:
+    - uses: actions/checkout@v3
+    - name: kubectl
+      run: curl -LO https://dl.k8s.io/release/${{matrix.kubectl_version}}/bin/linux/amd64/kubectl
+    - name: start from local
+      run: | 
+        export KIND_VERSION=${{matrix.kind_version}}
+        ./main.sh up
+    - name: verify
+      # todo: 
+      # - make verification in a shell script
+      # - make checking prometheus
+      # - make checking grafana
+      run: |
+        dpkg -l | grep bcc
+        docker ps -a
+        kind get kubeconfig --name=kind
+        kubectl get pods 
+
+# test local kind start up
+  test_localkind_with_prometheus:
+    runs-on: ubuntu-latest
+    needs: test_localkind
+    strategy:
+      matrix:
+        # I suppose we'd better update version pre kepler release.
+        # keep from current default version and latest version.
+        prometheus_operator_version: [v0.11.0, v0.12.0]
```

Comment:
Why do we need to test two prometheus version?

Reply:
in my point of view, we should keep two version been covered.
1st is current version we are using.
2nd is the latest version from prometheus operator.
hence once there any conflicts or issue with latest version of prometheus operator. we will know that at 1st time.
as this is PR level test, which means it running on latest branch.
before latest branch tagged with stable tag and further used in CI. we have enough time to fix it.

Reply:
> My concern is that running too many tests will slowdown the testing process.
don't worry, as kepler-action control which method been used in CI. hence this test just for code with in this repo.

yep, the version control will in kepler-action :-)